### ISSUE TITLE
Open prompt with semicolon instead of colon?

### ISSUE THREAD

**Issue by DLFW** [kamiyaa/joshuto#74]
I recognized that the prompt (aka command line) in joshuto can be opened with a semicolon while I would expect a colon. I first thought that it would be a bug, maybe triggered by my keyboard layout or so. However, when checking the code, I found this:
```rust
        let cmd = KeyCommand::CommandLine("".to_string(), "".to_string());
        let keys = [Event::Key(Key::Char(';'))];
        insert_keycommand(&mut m, cmd, &keys)?;
```
in `keymapping.rs`.

**So, my question is:** Is this by intention? If yes, why? Vim or ranger users would expect a colon to open the prompt, I guess. Since joshuto also shows a colon as a prompt, it confuses even more.

**Comment by LovecraftianHorror** [comment]
`ranger` also allows for opening the prompt using `;`

**Comment by DLFW** [comment]
Hm, true, just checked.  :smile: 
Ok, the question would be then why the colon is not supported in joshuto by default (additionally). I'll just file a PR...

**Comment by LovecraftianHorror** [comment]
That I would definitely agree with

**Comment by DLFW** [comment]
Issued #75

**Comment by kamiyaa** [comment]
Yeah, we can have both. I wasn't an extensive user of ranger's features so I will definitely have some features missing.

**Comment by kamiyaa** [comment]
Closed via #75
